pacemaker.spec.in

```diff
@@ -155,7 +155,7 @@ License:       GPLv2+ and LGPLv2+ and BSD
 Url:           http://www.clusterlabs.org
 Group:         System Environment/Daemons
 
-# eg. https://github.com/ClusterLabs/pacemaker/archive/8ae45302394b039fb098e150f156df29fc0cb576/pacemaker-8ae4530.tar.gz
+# Hint: use "spectool -s 0 pacemaker.spec" (rpmdevtools) to check the final URL
```

Comment:
The example URL allows someone to visually compare what the next line is trying to match, without having to do extra work. I'm fine with adding this hint above the example, but we should keep the example.